common/v2/features/SwapAssets/components/SwapAssets.tsx

```diff
@@ -160,8 +206,57 @@ export default function SwapAssets(props: Props) {
           onSelect={handleToAssetSelected}
           showOnlyTicker={true}
           disabled={isCalculatingToAmount || isCalculatingFromAmount}
+          searchable={true}
         />
       </FormItem>
+      <FormDisplay>
+        {initialRate && toAsset && fromAsset && (
+          <DisplayDataContainer>
+            <Label>{translateRaw('SWAP_RATE_LABEL')}</Label>
+            <DisplayData>
+              {translateRaw('SWAP_RATE_TEXT', {
+                $displayString: makeDisplayString(initialRate),
+                $toAssetSymbol: toAsset.symbol,
+                $fromAssetSymbol: fromAsset.symbol
+              })}
+            </DisplayData>
+          </DisplayDataContainer>
+        )}
+        {initialValue && toAmount && toAsset && (
+          <DisplayDataContainer>
+            <Label>
+              <LabelText>
+                {translateRaw('SWAP_FEE_LABEL', {
+                  $commission: MYC_DEXAG_COMMISSION_RATE.toString()
+                })}
+              </LabelText>
+              <STooltip tooltip={translateRaw('SWAP_FEE_TOOLTIP')}>
+                <img src={questionToolTip} />
+              </STooltip>
+              :
+            </Label>
+            <DisplayData>
+              {`${makeDisplayString(initialValue - parseFloat(toAmount))} ${toAsset.symbol}`}
```

Comment:
Both values can (over|under)flow BigNumber math please. e.g `.sub()`
Highly recommend to store values as big number and call `.toString()` when we need to display them.